### PR TITLE
Update accessibility.md

### PR DESCRIPTION
**Pull request by azhiya**
As agreed by the accessibility team, the existing Accessibility standards have been updated to provide better guidance.

### CODE REVIEW

File: wordpress-coding-standards/accessibility.md [hunk 1]
```diff
@@ -1,79 +1,129 @@
 # Accessibility Coding Standards
 
-These are standards that WordPress features should meet for accessibility in order to be merged into core. All new or updated code released in WordPress must conform with the WCAG 2.1 guidelines at level AA. These basic guidelines are intended for easy reference during development, but do not cover all possible accessibility issues.
+Code integrated into the WordPress ecosystem - including WordPress core, WordPress.org websites, and official plugins, is expected to conform to the Web Content Accessibility Guidelines, version 2.1, at level AA.
```
Comment (GaryJones) [inline wordpress-coding-standards/accessibility.md:3]:
```suggestion
Code integrated into the WordPress ecosystem - including WordPress core, WordPress.org websites, and official plugins, is expected to conform to the Web Content Accessibility Guidelines (WCAG), version 2.1, at level AA. 
```

Allows for using the abbreviation later on.

File: wordpress-coding-standards/accessibility.md [hunk 2]
```diff
@@ -1,79 +1,129 @@
...
-
-## Labeling
-
-Existing code uses a mixture of explicitly and implicitly labeled fields, but all new code must use an explicitly associated `<label>` element (using for/id attributes and _not_ wrapping the form control). Labels are not required to be visible, but must use the .screen-reader-text class when hidden. Placeholders are fine, but are not a substitute for labels. For all labels, clicking on the field label should cause the associated field to receive focus or, for checkboxes and radio selectors, select that choice.
-
-Don't introduce new title attributes to convey information. Use aria-label when you need to provide an alternate label and `.screen-reader-text` if you're appending additional data.
-
-When creating forms, use `<fieldset>` and `<legend>` to group logically related form elements inside complex forms or to group radio buttons and checkboxes under a heading.
+**Level A** success criteria address concerns considered to be first, accessibility barriers on a very wide scale that will prevent many people from accessing the site and, second, easily accomplished goals for the majority of web-based interfaces.
```
Comment (GaryJones) [inline wordpress-coding-standards/accessibility.md:25]: > **Level A** success criteria address concerns considered to be first

This doesn't make sense?

File: wordpress-coding-standards/accessibility.md [hunk 3]
```diff
@@ -1,79 +1,129 @@
...
+
+#### Principle: Robust
+**Guideline 4.1 Compatible**
+Maximize compatibility with current and future user agents, including assistive technologies.
+
+### Success Criteria
+Each guidance has a [specific list requirements that must be met for your content to be accessible](https://www.w3.org/WAI/WCAG21/quickref/). These tests can be carried out using automated software and or human testers. You can find more information on how to meet the success criteria in [Understanding Levels of Conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels). Whilst these criteria are important, usability testing is still important and should be carried out alongside any accessibility testing.
+
+### Techniques: Sufficient, Advisory, and Failures
+Techniques (code examples, resources, and tests) for guidance and success criteria that can help in making content more accessible, they are divided into three categories:
```
Comment (GaryJones) [inline wordpress-coding-standards/accessibility.md:97]:
```suggestion
Techniques (code examples, resources, and tests) for guidance and success criteria that can help in making content more accessible, are divided into three categories:
```

File: wordpress-coding-standards/accessibility.md [hunk 4]
```diff
@@ -1,79 +1,129 @@
...
+[Adrian Roselli](https://adrianroselli.com)
+[Scott O'Hara](https://www.scottohara.me)
+[Joe Dolson](https://www.joedolson.com/blog)
+[Sarah Higley](https://sarahmhigley.com/) 
+[Marco's Accessibility Blog](https://www.marcozehe.de/) 
+[Karl Groves](https://karlgroves.com/) 
+[Inclusive Components](https://inclusive-components.design) (Heydon Pickering)
+[Accessibility London (London, United Kingdom)](https://www.meetup.com/London-Accessibility-Meetup/) (London accessibility meetup: they live stream meetups on youtube)
+[24 Accessibility](https://www.24a11y.com/)
+[Mozilla Accessibility - Users first, no matter their abilities](https://blog.mozilla.org/accessibility/)
```
Comment (GaryJones) [inline wordpress-coding-standards/accessibility.md:119]:
```suggestion
- [WebAIM: Web Accessibility In Mind](https://webaim.org/) (see Articles and Resources)
- [Government Digital Service](https://gds.blog.gov.uk)
- [Accessibility in government](https://accessibility.blog.gov.uk/) 
- [Blog | TPG – The Accessibility Experts](https://developer.paciellogroup.com/blog/)
- [Web Accessibility Blog (Deque)](https://www.deque.com/blog/)
- [Tink - Léonie Watson](https://tink.uk) (Léonie Watson)
- [Adrian Roselli](https://adrianroselli.com)
- [Scott O'Hara](https://www.scottohara.me)
- [Joe Dolson](https://www.joedolson.com/blog)
- [Sarah Higley](https://sarahmhigley.com/) 
- [Marco's Accessibility Blog](https://www.marcozehe.de/) 
- [Karl Groves](https://karlgroves.com/) 
- [Inclusive Components](https://inclusive-components.design) (Heydon Pickering)
- [Accessibility London (London, United Kingdom)](https://www.meetup.com/London-Accessibility-Meetup/) (London accessibility meetup: they live stream meetups on youtube)
- [24 Accessibility](https://www.24a11y.com/)
- [Mozilla Accessibility - Users first, no matter their abilities](https://blog.mozilla.org/accessibility/)
```

Making it a list should make it easier for screen reader users to follow, than line-break-filled link after link after link.

File: wordpress-coding-standards/accessibility.md [hunk 5]
```diff
@@ -1,79 +1,129 @@
...
+
+### Technical and / or specific topics:
+[Accessibility Support](https://a11ysupport.io/) (Will your code work with assistive technologies?)
+[Accessibility APIs: A Key To Web Accessibility](https://www.smashingmagazine.com/2015/03/web-accessibility-with-accessibility-api/) (by Léonie Watson)  
+[How accessibility trees inform assistive tech](https://hacks.mozilla.org/2019/06/how-accessibility-trees-inform-assistive-tech/) (by Hidde de Vries) 
+[What is this thing and what does it do?](https://www.youtube.com/watch?v=YLihNhn_MO4 ) (presentation by Karl Groves)
+[The Browser Accessibility Tree](https://developer.paciellogroup.com/blog/2015/01/the-browser-accessibility-tree/) (by Steve Faulkner)
+[Brief history of browser accessibility support](https://www.paciellogroup.com/blog/2011/10/brief-history-of-browser-accessibility-support/) (by Steve Faulkner) 
+[ARIA Landmarks Example: General Principles](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/)
+[ARIA Landmarks Example: HTML Sectioning Elements](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html)
```
Comment (GaryJones) [inline wordpress-coding-standards/accessibility.md:129]:
```suggestion
- [Accessibility Support](https://a11ysupport.io/) (Will your code work with assistive technologies?)
- [Accessibility APIs: A Key To Web Accessibility](https://www.smashingmagazine.com/2015/03/web-accessibility-with-accessibility-api/) (by Léonie Watson)  
- [How accessibility trees inform assistive tech](https://hacks.mozilla.org/2019/06/how-accessibility-trees-inform-assistive-tech/) (by Hidde de Vries) 
- [What is this thing and what does it do?](https://www.youtube.com/watch?v=YLihNhn_MO4 ) (presentation by Karl Groves)
- [The Browser Accessibility Tree](https://developer.paciellogroup.com/blog/2015/01/the-browser-accessibility-tree/) (by Steve Faulkner)
- [Brief history of browser accessibility support](https://www.paciellogroup.com/blog/2011/10/brief-history-of-browser-accessibility-support/) (by Steve Faulkner) 
- [ARIA Landmarks Example: General Principles](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/)
- [ARIA Landmarks Example: HTML Sectioning Elements](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) 
```

**Review by GaryJones — CHANGES_REQUESTED** [review]
A few suggestions.

File: wordpress-coding-standards/accessibility.md (hunk 2 above)
Comment (azhiya) [reply wordpress-coding-standards/accessibility.md:25]: Hmmm, I see what you mean. How about
**Level A** success criteria address concerns considered to be accessibility barriers on a very wide scale that will prevent many people from accessing the site and the minimum set of accomplished goals required for the majority of web-based interfaces.

Comment (GaryJones) [reply wordpress-coding-standards/accessibility.md:25]: LGTM.

Comment (azhiya) [reply wordpress-coding-standards/accessibility.md:25]: @GaryJones I'm trying to change this but I don't think I'm accessing the right area.

Comment (GaryJones) [reply wordpress-coding-standards/accessibility.md]: @azhiya I've updated it.

**Comment by GaryJones** [comment]
@azhiya With the Level A description fixed, is this ready for review again?

**Comment by azhiya** [comment]
Yes, thank you @GaryJones

**Review by GaryJones — APPROVED** [review]

**Comment by azhiya** [comment]
Thank you @GaryJones

**Pull request merged**